build.gradle

```diff
@@ -62,19 +60,29 @@ ext {
   shouldUseDelombok = project.hasProperty('useDelombok')
 }
 
-String parseUiTestsTarget(String versionKey) {
-  if (versionKey =~ /^[0-9].*$/) {
-    return versionKey
-  } else {
-    def versionValue = intellijVersions.get(versionKey)
-    if (versionValue instanceof List) {
-      return versionValue.get(0)
+class ResolveIntelliJVersions extends DefaultTask {
```

Comment:
Workaround for running UI tests smoothly from CI (without a need to modify Circle CI config directly when bumping the versions)